frontend/src/components/MeetingRoom/ToolbarOverflowMenu/ToolbarOverflowMenu.tsx

```diff
@@ -47,6 +47,14 @@ const ToolbarOverflowMenu = ({
   const participantCount =
     subscriberWrappers.filter(({ isScreenshare }) => !isScreenshare).length + 1;
   const isReportIssueEnabled = import.meta.env.VITE_ENABLE_REPORT_ISSUE === 'true';
+
+  const handleCloseMenu = (onClick?: () => void) => () => {
```

Comment:
If you want to use a wrapper you could use a name like `closeMenuWrapper` to make it more obvious.